jobs/admin.py

```diff
@@ -11,10 +13,22 @@ class JobAdmin(admin.ModelAdmin):
             'fields': ('title', 'company', 'location'),
         }),
         ('Detay Bilgiler', {
-            'fields': ('url', 'date_created', 'description'),
+            'fields': ('is_closed', 'url', 'date_created', 'description'),
         }),
     )
 
-    list_display = ('title', 'company')
-    list_filter = ('company',)
+    list_display = ('title', 'company', 'is_closed')
+    list_filter = ('is_closed', 'company',)
     search_fields = ('title', 'company')
+
+    def get_fieldsets(self, request, obj=None):
```

Comment:
Henüz yerelimde denemediğim için soruyorum, neden bu metodu override etmek gerekiyor?